db/migrate/20150330221154_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :body
+      t.integer :rating
```

Comment:
watch your schema constraints here. They should match what's specified in the model. you should have `null: false` here